style.css

```diff
@@ -0,0 +1,72 @@
+.logo {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+  margin-left: 24px;
+}
+
+.menu {
+  display: flex;
+  margin-right: 24px;
+}
+
+.head-title {
+  margin: 12px 24px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+.content {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin: auto 24px;
+}
+
+.connect {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  margin: 12px 24px;
+}
+
+.fab,
+.far {
+  color: #505f79;
+  font-size: 20px;
+  margin-right: 20px;
+  display: inline;
+  float: left;
+}
+
+.icons {
+  min-width: 696px;
+  list-style: none;
+  display: table;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.background {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  background-image: url(image/header-shapes\ mobile.svg);
+  background-repeat: no-repeat;
+}
```

Comment:
- [ ] Please also as required in the project requirements, you must stick to the design as much as possible. It would be nice to add the fonts that are provided in Figma design to the logo, title, and description.